### ISSUE TITLE
BAM is converted to CRAM when running without fast5

### ISSUE THREAD

**Issue by ekushele** [epi2me-labs/wf-human-variation#91]
### Operating System

Ubuntu 22.04

### Workflow Version

v1.5.2

### Workflow Execution

Command line

### CLI command run

nextflow  run -c slurm_config.conf -resume /sci/home/mazaltov/.nextflow/assets/epi2me-labs/wf-human-variation/main.nf -profile singularity -params-file  params.yaml

Where params.yaml:
```
snp: true
sv: true
cnv: true
methyl: true
str: false

#Input Options
bam: bam/SARC07_bc04.bam
bam_min_coverage: 0.01
ref: links/genome.fa
sample_name: SARC07
out_dir: output_SARC07
```

### Workflow Execution - CLI Execution Profile

singularity

### What happened?

Somehow my bam was converted to cram, so the command `run_qdnaseq.r` fails

### Relevant log output

```shell
Caused by:
  Process `cnv:callCNV (1)` terminated with an error exit status (1)

Command executed:

  run_qdnaseq.r --bam SARC07.cram --out_prefix SARC07 --binsize 500 --reference hg38
  cut -f5 SARC07_calls.bed | paste SARC07_bins.bed - > SARC07_combined.bed
  check=`awk -F "       " 'NF != 6' SARC07_segs.seg`

  if [ -n "${check}" ]; then             echo "vcf is malformed";             workflow-glue fix_vcf --vcf SARC07_calls.vcf --fixed_vcf SARC07_calls_fixed.vcf --sample_id SARC07;             workflow-glue fix_vcf --vcf SARC07_segs.vcf --fixed_vcf SARC07_segs_fixed.vcf --sample_id SARC07;
      rm SARC07_calls.vcf
      mv SARC07_calls_fixed.vcf SARC07_calls.vcf
      rm SARC07_segs.vcf
      mv SARC07_segs_fixed.vcf SARC07_segs.vcf
  fi

Command exit status:
  1

Command output:
  (empty)

Command error:
  Loaded bin annotations for genome 'hg38', bin size 500 kbp, and experiment type 'SR50' from annotation package QDNAseq.hg38 v1.0.0
      SARC07.cram (1 of 1): extracting reads ...Error in value[[3L]](cond) :
    failed to open BamFile: 'filename' is not a BAM file
    file: /path/wf_human_variation/run_sarc_07/work/e6/4f46fc55a0c67afeb7d60c8df952a5/SARC07.cram
  Calls: binReadCounts ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
  Execution halted
```

**Comment by RenzoTale88** [comment]
@ekushele this has been fixed as of version v1.6.0 (see [here](https://github.com/epi2me-labs/wf-human-variation/releases/v1.6.0)).
Please, try running the workflow againg with the latest version (v.1.8.1). Also, please note that as of v1.8.0 we replaced the `--methyl` option with `--mod` (since I noticed you ask for methylation details). 
If you come across more issues with the latest release, please let us know in a new issue.
Best
Andrea

**Comment by ekushele** [comment]
Wow, that was fast!
I thought I will keep searching for answer untill I'll get reply, and I've just found out this is related to the version, but you replied before I could edit!

Thank you!
